code/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import styled from "styled-components/macro";
+import Loader from "react-loader-spinner";
+
+export const LoadingIndicator = () => {
+  const isLoading = useSelector((state) => state.game.isLoading);
+
+  return (
+    <Container>
+      {isLoading && (
+        <Loader
+          type="Oval"
+          color="#ff0000"
+          height={100}
+          width={100}
+          timeout={5000}
+        />
+      )}
+    </Container>
```

Comment:
Cool loader! I would like to see it either by itself or as it's own visual element, rather than on top of the content, though.